packages/vuetify/src/mixins/overlayable/index.ts

```diff
@@ -78,24 +78,29 @@ export default Vue.extend<Vue & Toggleable & Stackable & options>().extend({
 
       this.overlay = overlay
     },
+    declareOverlay () {
+      if (!this.overlay) return
+
+      if (this.activeZIndex !== undefined) {
+        this.overlay.zIndex = String(this.activeZIndex - 1)
+      } else if (this.$el) {
+        this.overlay.zIndex = getZIndex(this.$el)
+      }
+
+      this.overlay.value = true
+    },
     genOverlay () {
       this.hideScroll()
 
       if (this.hideOverlay) return
 
       if (!this.overlay) this.createOverlay()
 
-      requestAnimationFrame(() => {
-        if (!this.overlay) return
-
-        if (this.activeZIndex !== undefined) {
-          this.overlay.zIndex = String(this.activeZIndex - 1)
-        } else if (this.$el) {
-          this.overlay.zIndex = getZIndex(this.$el)
-        }
-
-        this.overlay.value = true
-      })
+      if (!document.hidden) {
+        requestAnimationFrame(this.declareOverlay())
```

Comment:
```suggestion
        requestAnimationFrame(this.declareOverlay)
```

Your current implementation will invoke immediately.

![image](https://user-images.githubusercontent.com/9064066/71867490-f3827c80-30cf-11ea-8e76-278a2e2825c4.png)